lib/data/models/todos_model.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:clean_code_architecture_flutter/data/models/todo_model.dart';
+
+class Todos {
+  int count;
+  List<TodoModel> data;
+
+  Todos({this.count, this.data});
+
+  Todos.fromJson(Map<String, dynamic> json) {
+    count = json['count'];
+    if (json['data'] != null) {
+      data = List<TodoModel>();
+      json['data'].forEach((v) {
```

Comment:
Lets have elaborate name instead of v
we can say json['data']. as `todos`
we can say v as `todo`.